### ISSUE TITLE
Please create a guide on how to make a TCP reverse tunnel for games like Minecraft

### ISSUE THREAD

**Issue by ImGGAAVVIINN** [fatedier/frp#4312]
### Describe the feature request

It would be very useful if a section in the README.md is dedicated to tunnelling for games and how to maximise the speed. Since the guides of TCP reverse tunnelling is spread across the long documentation and is hard to follow and under which protocol should be used to hosting (TCP Port Multiplexing OR TCP Stream Multiplexing)

### Describe alternatives you've considered

I tried to follow the tutorial, but are unable to succeed in tunnelling for a Minecraft server.

### Affected area

- [X] Docs
- [ ] Installation
- [ ] Performance and Scalability
- [ ] Security
- [X] User Experience
- [ ] Test and Release
- [ ] Developer Infrastructure
- [ ] Client Plugin
- [ ] Server Plugin
- [ ] Extensions
- [ ] Others

**Comment by fatedier** [comment]
We currently do not have such a plan. If there is enough time in the future, the document content will be migrated to an independent document site, which may be more suitable for such content.
